### ISSUE TITLE
GCop 514

### ISSUE THREAD

**Issue by marjanjavid** [Geeksltd/GCop#157]
Please Provide a sample for gcop514
As it can be seen in the picture below, this rule is not kicked for this sample
![514](https://user-images.githubusercontent.com/20684649/40575886-9ae6c494-6102-11e8-95ff-07817ca679cb.PNG)

**Comment by Foroughi** [comment]
@marjanjavid  Would you please provide the complete code including the namespace and its class ?! I just checked your code and it worked for me. So would you please provide the complete code here ?! 

Unit Test has been added on [Development\a962445daad54c49c43e61146a8c041fc9c368bf](https://gitlab.com/Paymon/GCop/tree/a962445daad54c49c43e61146a8c041fc9c368bf)

You may wanna check the unit test i`ve added

**Comment by marjanjavid** [comment]
@Foroughi It seems I have written this in a BaseView classs
![12](https://user-images.githubusercontent.com/20684649/42220298-c6bee732-7ee3-11e8-83b4-4a68a3fa1fd0.PNG)

**Comment by Foroughi** [comment]
@marjanjavid  we need to talk about this as the rule is kicking in for the same code on my end.

**Comment by marjanjavid** [comment]
@Foroughi Yes I run the unit test and it works fine. weird in my code it doesn't do anything. I test it on GCop2.6.0. @Foroughi would you please check it on a sample project?
